inst/draggable-buckets/draggable-buckets.js

```diff
@@ -90,7 +90,7 @@ $.extend(draggableBuckets, {
   subscribe: function (el, callback) {
     if (this.observers === undefined) this.observers = {};
     this.observers[el] = new MutationObserver(function () {
-      callback({ priority: "event" });
+      callback({ priority: "deferred" });
```

Comment:
Good catch. Previously we called the callback function with a mutationsList which got type casted to boolean as true.
So, now we need to use "deferred" as mentioned [here](https://github.com/rstudio/shiny/blob/6325067130dcf4cdd62aa2ffa59b71d1be3af960/srcts/src/bindings/input/inputBinding.ts#L9-L16).

Although, I'm not so sure about vbumping the shiny version because even in older shiny versions which only support boolean values in the callback, the value { priority: "deferred" } is going to be type casted into true. Let me test this branch with older shiny version 1.6.0 to be sure. @averissimo What are your thoughts to just make it "deferred" without vbumping?

Reply:
AFAIK, `teal` is not present in validated repositories, but I understand your point.

I'll make the change back and we can revisit the topic once `shiny` has a few more releases